### DISCUSSION TITLE
Page refresh issue ?id= to ?path=

### DISCUSSION THREAD

**Discussion by BartJanvanAssen** [storybookjs/storybook#12773] [Help]
Hey, 

when I'm refreshing my page I will always be redirected to the first story, and I see the URL change from {host}?id= to {host}?path={path of the first story}

options: `
addParameters({
  options: {
    name: '',
    url: 'https://example.com/',
    goFullScreen: false,
    showAddonsPanel: true,
    showSearchBox: false,
    addonPanelInRight: true,
    sortStoriesByKind: true,
    hierarchySeparator: /\./,
    hierarchyRootSeparator: /\|/,
    enableShortcuts: true
  },
  backgrounds: [
    { name: 'background', value: '#f6f6f6', default: true },
    { name: 'AppBar', value: '#FFFFFF' }
  ]
})
`

Any suggestion? it really frustrates me... can't seem to find our why or find anything about it.
I'm on version 6 of SB

**Comment by shilman** [comment]
haven't seen that. can you post a screen capture of the repro? i think the URL should always be `path=`
